### PR TITLE
Maps

### PR DESCRIPTION
**Pull request by jamesmahler2**
- added "utf-8" encoding to some of the Twitter files cause I was getting an encoding error
- made the two map plots side-by-side
- if user selects a daily statistic, then show a time series for each county (or user can select which counties to display)
- if user selects a cumulative statistic, then plot the statistic vs the economic/educational statistic to show the interaction between the two

### CODE REVIEW

**Comment by lee-vivian** [comment]
This looks great! 

One thing I noticed was that (for the Unemployment and Median HHI features only) the correlation map between the USDA features and Cumulative Covid stats appears empty. Is that supposed to happen?

But overall I agree - the app looks really nice right now. If the observation above is a bug and is more complicated to fix, we could merge this now and create a new issue for it. Whichever is easier/you'd prefer. Thanks!

Screen Shot 2020-11-18 at 12 07 33 PM

File: data/tweets/geo_covid_tweet_ids/geo/sc.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 1260291666130489346
+1260291666130489346
```
Comment (jkosh44) [inline data/tweets/geo_covid_tweet_ids/geo/sc.txt]: Was this added by accident?

**Review by jkosh44 — APPROVED** [review]

File: data/tweets/geo_covid_tweet_ids/geo/sc.txt (hunk 1 above)
Comment (jamesmahler2) [reply data/tweets/geo_covid_tweet_ids/geo/sc.txt]: Yeah, not sure how it happened. Will delete the file.

**Comment by jamesmahler2** [comment]
Good catch @lee-vivian it looks like for some of the dataframes the area name is in the form of "<countyName>,<stateName>", and for others it's in the form of "<countyName>". I'll join the two dataframes on the FIPS variable instead and hopefully we won't have any more issues.

**Pull request merged**